### PR TITLE
resource/alicloud_polardb_cluster: modified support loose_xengine and loose_xengine_use_memory_pct.

### PR DESCRIPTION
**Pull request by mengliux**
resource/alicloud_polardb_cluster: modified support loose_xengine etc; resource/alicloud_polardb_cluster_test: modified support loose_xengine etc; docs: support loose_xengine etc.

### CODE REVIEW

**Comment by MrWolong** [comment]
这个aone增加了两个字段，pr标题改一下，参考pr7722，标题中要把新增的两个字段都体现出来

File: alicloud/resource_alicloud_polardb_cluster_test.go [hunk 1]
```diff
@@ -693,6 +693,7 @@ func TestAccAliCloudPolarDBCluster_CreateNormal(t *testing.T) {
 					"parameter_group_id":     "${data.alicloud_polardb_parameter_groups.default.groups.0.id}",
 					"lower_case_table_names": "0",
 					"backup_retention_policy_on_cluster_deletion": "NONE",
+					"loose_xengine": "OFF",
```
Comment (MrWolong) [inline alicloud/resource_alicloud_polardb_cluster_test.go:696]: 单测只覆盖了创建时指定loose_xengine。新增一个单测，创建时同时指定loose_xengine和loose_xengine_use_memory_pct

File: website/docs/r/polardb_cluster.html.markdown [hunk 2]
```diff
@@ -207,6 +207,10 @@ The following arguments are supported:
   -> **NOTE:** This parameter is valid only for standard edition clusters.
 * `loose_polar_log_bin` - (Optional, Computed, Available since 1.210.0) Enable the Binlog function. Default value: `OFF`. Valid values are `OFF`, `ON`.
   -> **NOTE:** This parameter is valid only MySQL Engine supports.
+* `loose_xengine` - (Optional, Available since v1.231.1) Specifies whether to enable X-Engine. Valid values are `ON`, `OFF`.
+  -> **NOTE:** This parameter takes effect only if you do not set CreationOption to CreateGdnStandby and you set DBType to MySQL and DBVersion to 8.0. To enable X-Engine on a node, make sure that the memory of the node is greater than or equal to 8 GB in size.
```
Comment (MrWolong) [inline website/docs/r/polardb_cluster.html.markdown:211]: -> **NOTE:** This parameter takes effect only if you do not set CreationOption to CreateGdnStandby and you set DBType to MySQL and DBVersion to 8.0. To enable X-Engine on a node, make sure that the memory of the node is greater than or equal to 8 GB in size.
把这个描述中涉及到的属性都改成tf中的字段名，如DBType应该写成`db_type`，字段应该被"``"包裹

File: website/docs/r/polardb_cluster.html.markdown [hunk 3]
```diff
@@ -207,6 +207,10 @@ The following arguments are supported:
...
+* `loose_xengine_use_memory_pct` - (Optional, Available since v1.231.1) Set the ratio to enable the X-Engine storage engine. Valid values: 10 to 90.
+  -> **NOTE:** When the parameter loose_xengine is ON, it takes effect.
```
Comment (MrWolong) [inline website/docs/r/polardb_cluster.html.markdown:213]: 把it改成要指定字段名

File: alicloud/resource_alicloud_polardb_cluster.go [hunk 4]
```diff
@@ -440,6 +440,20 @@ func resourceAlicloudPolarDBCluster() *schema.Resource {
 					"+12:00", "+13:00", "SYSTEM"}, false),
 				DiffSuppressFunc: polardbDiffSuppressFunc,
 			},
+			"loose_xengine": {
```
[the remark this answers is not in the document]
Comment (mengliux) [reply alicloud/resource_alicloud_polardb_cluster.go:443]: loose_xengine跟压制函数字段同时满足才有效，不满足就会被忽略，loose_xengine_use_memory_pct压制函数去了，压制函数跟文档已保持一致了

File: alicloud/resource_alicloud_polardb_cluster_test.go (hunk 1 above)
Comment (mengliux) [reply alicloud/resource_alicloud_polardb_cluster_test.go:700]: 已加，单测结果在测试报告中贴着

File: website/docs/r/polardb_cluster.html.markdown (hunk 2 above)
Comment (mengliux) [reply website/docs/r/polardb_cluster.html.markdown]: 已修改

File: website/docs/r/polardb_cluster.html.markdown (hunk 3 above)
Comment (mengliux) [reply website/docs/r/polardb_cluster.html.markdown]: OK

**Review by MrWolong — COMMENTED** [review]
approved

**Comment by mengliux** [comment]
image
image

**Pull request merged**